profile-widget/diveeventitem.cpp

```diff
@@ -152,17 +152,21 @@ void DiveEventItem::setupToolTipString()
 
 		/* Do we have an explicit cylinder index?  Show it. */
 		if (internalEvent->gas.index >= 0)
-			name += QString(" (cyl %1)").arg(internalEvent->gas.index+1);
+			name += tr(" (cyl.") + QString(" %1)").arg(internalEvent->gas.index+1);
```

Comment:
Having the argument in the translated string is actually almost required, in case you have a target  language that needs to be able to re-order the words. Admittedly, that's less likely with a trivial text like this, but even with a single word this can happen:
Mr. %s
Herr %s
%s xiānshēng (先生)